src/js/ripple/log.js

```diff
@@ -149,7 +149,11 @@ var NullLogEngine = {
 if (typeof window !== 'undefined' && typeof console !== 'undefined') {
   Log.engine = InteractiveLogEngine;
 } else if (typeof console !== 'undefined' && console.log) {
-  Log.engine = BasicLogEngine;
+  if (process.env.NODE_ENV === 'test') {
+    Log.engine = NullLogEngine;
```

Comment:
I agree with the first part, but why do you think we should suppress all error messages? We wouldn't have even noticed the problem that started this task if we were suppressing everything.